database/index.js

```diff
@@ -6,7 +6,7 @@ mongoose.Promise = global.Promise
 module.exports = {
   connect: () => {
     mongoose
-      .connect(config.local_url, {
+      .connect(config.mlab_url, {
```

Comment:
Why? I added this so it tries to connect to a local instance first, if it fails will connect to mLab.

Reply:
There is a lot of data in the local database.